xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-api/src/main/java/org/xwiki/search/solr/AbstractSolrCoreInitializer.java

```diff
@@ -138,25 +143,34 @@ public abstract class AbstractSolrCoreInitializer implements SolrCoreInitializer
 
     private static final String SOLR_VERSIONFIELDTYPE_VALUE = "defVal";
 
+    private static final int MIGRATION_BATCH_ROWS = 100;
```

Comment:
This seems quite low, couldn't we set this to a higher value like 1000? Or do we really have Solr cores with large documents?

Reply:
Indeed, the cores which are actually going to be migrated in XWiki Standard are not supposed to have very big document, but it might be the case later or in some extension.

To be safe, I'm going to keep 100 by default and allow AbstractSolrCoreInitializer to provide their own values (could even be more than 1000 for ratings for example, since those are supposed to be very small documents).